pkg/server/config.go

```diff
@@ -18,7 +18,15 @@ type (
 		GRPCPort string `mapstructure:"grpc_port"`
 		HTTPPort string `mapstructure:"http_port"`
 
-		Cors Cors `mapstructure:"cors"`
+		Timeouts Timeouts `mapstructure:"timeouts"`
```

Comment:
Maybe is a good idea to indicate here that we are providing the `http` timeouts. Because historically, we have mixed settings here for `grpc` and `http`. Something like
```suggestion
		HTTPTimeouts HTTPTimeouts `mapstructure:"http_timeouts"`
```
wdyt?

Reply:
done